packages/cloudflare/src/index.ts

```diff
@@ -191,16 +155,15 @@ export default function createIntegration(args?: Options): AstroIntegration {
 							isSharpCompatible: false,
 							isSquooshCompatible: false,
 						},
+						envGetSecret: 'experimental',
 					},
 				});
 			},
 			'astro:server:setup': async ({ server }) => {
 				if (args?.platformProxy?.enabled === true) {
```

Comment:
I need to fix this condition, to have a default value of `true`

Reply:
ah yeah my bad when I refactored I messed up that part